docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -123,16 +123,23 @@ done
+<1> The source cloud `default` cell acquires a new `DEFAULT_CELL_NAME` on the destination cloud after adoption.
+In a multi-cell adoption scenario, you can retain the original name, `default`, or create a new cell default name, for example, `cell<X>`, by providing the incremented index of the last cell in the source cloud (which is, by adding a 1 to it).
+//Bohdan: Can you clarify what you mean by "adding a 1 to it"? Do you mean that if the last cell in the source cloud was cell5, then you would add a 1 and it would be cell6?
+<2> For each cell, update the <["standalone.localdomain"]="x.x.x.x"> and complete `COMPUTES_CELL<X>` and `COMPUTES_API_CELL<X>` data with the names and IP addresses of the {compute_service} nodes that are connected to the `ctlplane` and `internalapi` networks. Do not specify a real FQDN defined for each network, always use the same host name for each of its connected networks. Providing IP addresses and names of the hosts on remaining networks of the source cloud should be covered the similar way. Or you can adjust the generated files manually as shown below.
+//Bohdan: Are customers supposed to replace the COMPUTES_CELL1, COMPUTES_CELL2, COMPUTES_API_CELL1, etc. with the names of their own compute cells? Or do you mean that customers should replace ["standalone.localdomain"]="192.168.122.100 with the name and IP address of the Compute nodes?
+<3> If your deployment has a custom DNS domain, specify it in the FQDN value of the nodes. The given values are used in the data plane node set `spec.nodes.<NODE NAME>.hostName`.
+//Bohdan: This point corresponds with `["standalone.localdomain"]="192.168.122.100"` in the code block, and I'm not sure how they relate.Also, does the "given values" refer to the custom DNS domain?
+<4> Assign all {compute_service} nodes from the source cloud `cell1` cell into `COMPUTES_CELL1`, and so on.
+//Bohdan: <4> points to #... in the code block. Is the customer expected to list the Compute nodes from `cell1` there?
 <5> Assign all {compute_service} nodes from the source cloud `default` cell into `openstack-<X>`,
 where `<X>` is the `DEFAULT_CELL_NAME` environment variable value (here, it equals 'cell3').
-<6> Cells not containing compute nodes will be omitted as no node sets for it should be created.
+//Bohdan:So in this example, `cell3` is considered the `DEFAULT_CELL_NAME`? And `openstack-<X>` is what the `default` cell was called in the source cloud?
+<6> Cells that do not contain Compute nodes are omitted because no node sets are created for the cells.
+//Bohdan: <6> in the code block should probably be moved after `NODESETS="'openstack-${CELL}', $NODESETS"`, correct? And can we also describe what is happening in that command? For example: "<6> Creates the node sets for each cell that include Compute nodes. Cells that do not include Compute nodes are omitted because no node sets are created for the cells.
```

Comment:
It could be moved down as well, yes.
You can rephrase like that, but a better word for this code block it tepmplated, not really created yet